test/config-luatest/rpc_test.lua

```diff
@@ -419,6 +419,91 @@ g.test_filter_mode = function(g)
     g.server_4:exec(check)
 end
 
+g.test_filter_works_in_parallel = function(g)
```

Comment:
AFAIU, the test case works ~10 seconds. That's generally bad (but understandable here). I think that it worth to move such long test cases into its own file to be sure that the default test timeout limit (110 seconds) is not reached even in a slow environment.

OTOH, other test cases are likely fast and unlikely there is a real problem with it. I'm in a little doubt, so just noted it. Feel free to ignore.

Reply:
I like your idea with `connect_timeout` option. Can you create an issue for that, so we don't loose it.